src/HDF5.jl

```diff
@@ -931,7 +931,13 @@ function g_create(f::Function, parent::Union{HDF5File, HDF5Group}, args...)
     end
 end
 
-function d_create(parent::Union{HDF5File, HDF5Group}, path::String, dtype::HDF5Datatype, dspace::HDF5Dataspace, lcpl::HDF5Properties, dcpl::HDF5Properties, dapl::HDF5Properties, dxpl::HDF5Properties)
+# N.B. lcpl does not have a default to distinguish it from the no-keyword method of the
+#      following method.
```

Comment:
```suggestion
# NOTE: lcpl does not have a default to distinguish it from the no-keyword method of the following method.
```

Reply:
Ah, stepping away made me realize there's another way around this — just dispatch through another private `_d_create` that sets the default properties, and that hands off to the true low-level `d_create`.

In looking at this, though, I've noticed `a_create` never actually accepts keywords/properties, even though these higher-level wrapper suggest one could use them. Should that lie be maintained?